metrics/lib/features/dashboard/domain/entities/core/build.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:equatable/equatable.dart';
+import 'package:flutter/foundation.dart';
+import 'package:metrics/features/dashboard/domain/entities/core/percent.dart';
+
+enum BuildStatus { successful, cancelled, failed }
+
+/// Represents the build entity.
```

Comment:
What if we improve documentation a bit more to add more details on what Build actually means?